Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -664,6 +664,45 @@ extension BrowserViewController: WKNavigationDelegate {
     guard let tab = tab(for: webView), let url = webView.url, rewards.isEnabled else { return }
     tab.redirectURLs.append(url)
   }
+  
+  /// Invoked when an error occurs while starting to load data for the main frame.
+  public func webView(_ webView: WKWebView, didFailProvisionalNavigation navigation: WKNavigation!, withError error: Error) {
+    // Ignore the "Frame load interrupted" error that is triggered when we cancel a request
+    // to open an external application and hand it over to UIApplication.openURL(). The result
+    // will be that we switch to the external app, for example the app store, while keeping the
+    // original web page in the tab instead of replacing it with an error page.
+    let error = error as NSError
+    if error.domain == "WebKitErrorDomain" && error.code == 102 {
+      return
+    }
+
+    if checkIfWebContentProcessHasCrashed(webView, error: error as NSError) {
```

Comment:
you have a `let error = error as NSError` just above. Do you need that `as NSError` here?